arithmetization/src/main/java/net/consensys/linea/zktracer/module/mmio/Mmio.java

```diff
@@ -55,31 +58,37 @@ public String moduleKey() {
   }
 
   @Override
-  public void enterTransaction() {}
+  public void enterTransaction() {
+    lineCounter.enter();
+  }
 
   @Override
-  public void popTransaction() {}
+  public void popTransaction() {
+    lineCounter.pop();
+  }
```

Comment:
There seems to be a lot of repetition wrt the `enterTransaction` and `popTransaction` methods. Maybe we could have a default implementation that they inherit ?

Reply:
done, I now introduce two specalised interface, one for StatelessModule (ADD, WCP, etc), and one for normal module (RLP_*, etc)